scripts/overlap/kubectl-kubestellar-create-space

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: $0 (--space-name space-name | --provider-name provider-name | --sm-core-config sm-kubeconfig | -X)* 
+
+# Purpose: creates a space in the space framework.
+
+
+bindir="$(dirname "$0")"
+
+provider_name="default"
+
+while (( $# > 0 )); do
+    case "$1" in
+	(-h|--help)
+	    echo "Usage: kubectl kubestellar get-config-for-space (--space-name name | --sm-core-config filepath "
+	    exit 0;;
+	(-X) set -o xtrace;;
+	(--space-name)
+	    if (( $# >1 ))
+	    then space_name="$2"; shift
+	    else echo "$0: missing space name" >&2; exit 1
+	    fi;;
+	(--provider-name)
+	    if (( $# >1 ))
+	    then provider_name="$2"; shift
+	    else echo "$0: missing space provider name" >&2; exit 1
+	    fi;;	
+	(--sm-core-config)
```

Comment:
A similar comment to https://github.com/kubestellar/kubestellar/pull/1268/files#r1411549630 applies here. I think that for this script the obvious approach is to have its command line flags and `$KUBECONFIG` point to the space that holds the space management API objects.